lib/mapObjects/JsonRandom.cpp

```diff
@@ -104,16 +114,40 @@ namespace JsonRandom
 		{
 			ret.push_back(loadValue(value[name], rng));
 		}
+		
+		if(!value["random"].isNull())
```

Comment:
Can probably get same treatment as resources/secondaries, for consistency

Reply:
Agree, done